features/support/ui/landing.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'ui'
+
+module UI
+  class Landing < SitePrism::Page
+    set_url '/'
+    set_url_matcher(/wpcc\/\d+\/landing/)
+
+    element :age, '#your_details_form_age'
```

Comment:
In order to de-couple tests from actual CSS implementation we usually precede non-styling class names with `t-`. The elements can be changed to:

```
element :age, '.t-age_field'
element :gender, '.t-gender_field'
element :salary, '.t-salary_field'
```